runtime/ceylon/language/Character.java

```diff
@@ -321,7 +321,7 @@ public boolean notLargerThan(@Name("other") Character other) {
 
     public static int codepoint(long value) {
         if (value>0x10FFFF || value<0) {
-            throw new OverflowException(value + " is not a 32-bit integer");
+            throw new OverflowException(value + " is not a possible Unicode code point");
```

Comment:
Here the original message was simply wrong (i.e. this could happen even if the value was a 32-bit integer).